### ISSUE TITLE
process-local.py + python3 = JSON issues

### ISSUE THREAD

**Issue by jasonmayes** [affinelayer/pix2pix-tensorflow#16]
I can not seem to get process-local.py working with python3. I admit I am new to Python, but it seems to be something to do with the fact the strings are treated as binary eg b'....'  so the JSON encoding fails. 

Any ideas here what to do to tweak this code to get it running? Everything else for the training / testing works fine with p3.

Thanks for the great work by the way, had a lot of fun trying it out!

**Comment by jasonmayes** [comment]
Update: I couldn't figure this out so I have rolled back to Python 2.7 and it is working with 2.7. Quick question, it seems it generates exactly the same image every time I run it on the same input, in the demo it differs each time you run it, how do I enable this method of generation?

**Comment by christopherhesse** [comment]
This should be fixed in the next release later today.  Apparently making things work in both py 2.7 and 3.6 is a non-trivial thing.

**Comment by jasonmayes** [comment]
Wonderful thank you so much!

**Comment by christopherhesse** [comment]
Generating different images was not intentional, and I don't think they are substantially different in my experience.  I suspect that the noise is from the use of dropout at inference time.
